### PR TITLE
My capstone project

### PR DESCRIPTION
**Pull request by dami1080**
In this pull request, I  have; 

- added an HTML file
- added a CSS file
- all linters are now passing
- added readme file
- an app.js file

           **The following was implemented**

- Create a javascript array of objects to store speakers
 - Populate the speakers dynamically on page load
- Add media queries for screens below 768px below and 768px and above
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
- When the hamburger button on the header is clicked, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                </div>
+                <p class="rights">
+                    @ 2021 BLOCKCHAIN SUMMIT. A Some Rights Reserved
+                </p>
+            </div>
+        </div>
+    </footer>
+    <script src="app.js"></script> 
+</body>
+</html>
```
Comment (de-scholar) [inline about.html]: - [ ] There should be at least 1, but no more than 1, h1 tag in each page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                </div>
+                <p class="rights">
+                    @ 2021 BLOCKCHAIN SUMMIT.
+                </p>
+            </div>
+        </div>
+    </footer>
+    <script src="app.js"></script>
+</body>
+</html>
```
Comment (de-scholar) [inline index.html]: - [ ] There's at least 1, but no more than 1, h1 tag in each page.
- [ ] The page is accessible, images contain alt attributes and there's not images or elements blocking the text in both screen sizes.

**Review by de-scholar — CHANGES_REQUESTED** [review]
Hi @dami1080 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] The video should contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- [ ] The video should contain a highlight about a piece of the code the student finds interesting or is proud of making.

:point_right: I suggest you record another video which fulfill all of the video presentation requirements! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                 Commons affliate network in a different country
+                 every two years. This year, we hope to expand our invitation list
+                 including organizations and individuals
+                </p>
+                <div class="hero-about-content-contact">
+                    <p>
+                       Please contact us per Email for any
+                       further questions about Block Chain Summit
+                       2021!
+                    </p>
```
Comment (ElijahTobs) [inline about.html]: - [ ] As the previous reviewer stated, there should be one but not more than one `h1` tag on each page. The `about.html` page currently does not have an `h1` tag

File: about.html [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+                    <h2 class="summits-heading">
+                        See the past Block Chain Summits
+                    </h2>
+                    <p class="summits-text">
+                        Take a look at the last two Block Chain
+                        Summit which took place in China and in Dubai
+                    </p>
+                    <ul class="summits-list">
+                        <li class="summits-list-item">
+                            <img src="./images/conference_one.jpg" alt="">
```
Comment (ElijahTobs) [inline about.html]: - [ ] The previous reviewer also mentioned the alt attribute but it has not been addressed. The alt tag describes the image or at least states what the image is trying to convey to the person seeing the page. In addition, users of screen magnification may not be able to get a complete view of the image and thus need the alt tag to let them know what the purpose of the image is. The same goes for `line 98`

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @dami1080  👋

You've done really well so far! 👍 

## Major Highlights 💯 

Your README is professional ✔️ 
You've followed the Github flow rules. ✔️ 
You have good commit messages ✔️ 
Nice Pull Request description ✔️ 
Linter checks are passing ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

__N/A__

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @dami1080 , 

Good job making the required changes.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**